examples/tree_2d_dgsem/elixir_navierstokes_convergence.jl

```diff
@@ -8,163 +8,170 @@ prandtl_number() = 0.72
+             (2.0 * A * exp(-A * (y - 1.0)) / (y + 2.0) -
+              A * A * log(y + 2.0) * exp(-A * (y - 1.0)) -
+              (1.0 - exp(-A * (y - 1.0))) / ((y + 2.0) * (y + 2.0))) * cos(pi_t))
+    v2 = v1
+    v2_t = v1_t
+    v2_x = v1_x
+    v2_y = v1_y
+    v2_xx = v1_xx
+    v2_xy = v1_xy
+    v2_yy = v1_yy
+
+    p = rho * rho
+    p_t = 2.0 * rho * rho_t
+    p_x = 2.0 * rho * rho_x
+    p_y = 2.0 * rho * rho_y
+    p_xx = 2.0 * rho * rho_xx + 2.0 * rho_x * rho_x
+    p_yy = 2.0 * rho * rho_yy + 2.0 * rho_y * rho_y
+
+    # Note this simplifies slightly because the ansatz assumes that v1 = v2
+    E = p * inv_gamma_minus_one + 0.5 * rho * (v1^2 + v2^2)
+    E_t = p_t * inv_gamma_minus_one + rho_t * v1^2 + 2.0 * rho * v1 * v1_t
+    E_x = p_x * inv_gamma_minus_one + rho_x * v1^2 + 2.0 * rho * v1 * v1_x
+    E_y = p_y * inv_gamma_minus_one + rho_y * v1^2 + 2.0 * rho * v1 * v1_y
+
+    # Some convenience constants
+    T_const = equations.gamma * inv_gamma_minus_one / Pr
+    inv_rho_cubed = 1.0 / (rho^3)
+
+    # compute the source terms
+    # density equation
+    du1 = rho_t + rho_x * v1 + rho * v1_x + rho_y * v2 + rho * v2_y
+
+    # x-momentum equation
+    du2 = (rho_t * v1 + rho * v1_t + p_x + rho_x * v1^2
+           + 2.0 * rho * v1 * v1_x
+           + rho_y * v1 * v2
+           + rho * v1_y * v2
+           + rho * v1 * v2_y -
+           # stress tensor from x-direction
+           4.0 / 3.0 * v1_xx * mu_ +
+           2.0 / 3.0 * v2_xy * mu_ -
+           v1_yy * mu_ -
+           v2_xy * mu_)
+    # y-momentum equation
+    du3 = (rho_t * v2 + rho * v2_t + p_y + rho_x * v1 * v2
+           + rho * v1_x * v2
+           + rho * v1 * v2_x
+           + rho_y * v2^2
+           + 2.0 * rho * v2 * v2_y -
+           # stress tensor from y-direction
+           v1_xy * mu_ -
+           v2_xx * mu_ -
+           4.0 / 3.0 * v2_yy * mu_ +
+           2.0 / 3.0 * v1_xy * mu_)
+    # total energy equation
+    du4 = (E_t + v1_x * (E + p) + v1 * (E_x + p_x)
+           + v2_y * (E + p) + v2 * (E_y + p_y) -
+           # stress tensor and temperature gradient terms from x-direction
+           4.0 / 3.0 * v1_xx * v1 * mu_ +
+           2.0 / 3.0 * v2_xy * v1 * mu_ -
+           4.0 / 3.0 * v1_x * v1_x * mu_ +
+           2.0 / 3.0 * v2_y * v1_x * mu_ -
+           v1_xy * v2 * mu_ -
+           v2_xx * v2 * mu_ -
+           v1_y * v2_x * mu_ -
+           v2_x * v2_x * mu_ -
+           T_const * inv_rho_cubed *
+           (p_xx * rho * rho -
+            2.0 * p_x * rho * rho_x +
+            2.0 * p * rho_x * rho_x -
+            p * rho * rho_xx) * mu_ -
+           # stress tensor and temperature gradient terms from y-direction
+           v1_yy * v1 * mu_ -
+           v2_xy * v1 * mu_ -
+           v2_x * v1_y * mu_ -
+           4.0 / 3.0 * v2_yy * v2 * mu_ +
+           2.0 / 3.0 * v1_xy -
+           4.0 / 3.0 * v2_y * v2_y * mu_ +
+           2.0 / 3.0 * v1_x * v2_y * mu_
+           -
+           -
```

Comment:
Formatting appears to have removed something crucial

Reply:
I don't think the formatter did that, but the manual formatting, e.g., [here](https://github.com/trixi-framework/Trixi.jl/pull/1531/commits/78831ef82beb287042102eb272295b7b6e527bf7?diff=unified&w=0#diff-aefb6b57a33d9112f7244d28e7fb1b7ee8d40e2076e1c7e7570e65681381486fR163).